doc/Analytics.xml

```diff
@@ -3496,6 +3494,69 @@
     </tt:ParentTopic>
   </tt:Messages>
 </tt:RuleDescription>
+]]></programlisting>
+    </section>
+    <section>
+      <title>Occupancy Counting rule</title>
+      <para>The occupancy counting rule counts the number of objects inside an enclosure by counting
+        objects passing in and out of a set of line segments (barriers). The counter increments when
+        objects pass from left to right and decrements when objects pass from right to left.</para>
+      <para>The configuration parameters also include the time interval to report the events and 
+        time interval to reset its counter.</para>
+      <variablelist role="op">
+        <varlistentry>
+          <term>Parameters</term>
+          <listitem>
+            <para role="param">Segments - unbounded [tt:Polyline]</para>
+            <para role="text">One or more polylines used for detection. </para>
+            <para role="param">ReportTimeInterval - optional [xs:duration]</para>
+            <para role="text">Optional time interval to reduce number of reported changes by
+              reporting aggregated values.</para>
+            <para role="param">ResetTime - optional, unbounded [xs:time]</para>
+            <para role="text">Time or times of the day when the counter should be reset.
+                <emphasis>Question: localtime or utc?</emphasis></para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Topic</term>
+          <listitem>
+            <para role="param">tns1:RuleEngine/CountAggregation/OccupancyCounter</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Source</term>
+          <listitem>
+            <para role="text">See <xref linkend="_msgSource"/></para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Data</term>
+          <listitem>
+            <para role="param">Count [xs:int]</para>
+            <para role="text">Number of objects counted since last reset.</para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+      <para>CountAggregation defined by the following code using the rule description language: </para>
+      <programlisting><![CDATA[<tt:RuleDescription Name="tt:OccupancyCounting">
+  <tt:Parameters>
+    <tt:ElementItemDescription Name="Segments" Type="tt:Polyline"/>
+    <tt:SimpleItemDescription Name="ReportTimeInterval" Type="xs:duration"/>
+    <tt:SimpleItemDescription Name="ResetTime" Type="xs:time"/>
+    <tt:SimpleItemDescription Name="Direction" Type="tt:Direction"/>
```

Comment:
In occupancy counter direction parameter is not required, since its already defined above which direction it will be incremented and decremented. Please check.

Reply:
Request to update the header for the section A.5 as **A.5 Line crossing counting rule** as line counting rule is a bit incomplete.